superset-frontend/cypress-base/cypress/integration/explore/control.test.ts

```diff
@@ -233,8 +233,158 @@ describe('Time range filter', () => {
       });
     cy.get('[data-test=cancel-button]').click();
   });
-});
+  it('Chosen custom time range parameters are applied after change', () => {
+    const formData = {
+      ...FORM_DATA_DEFAULTS,
+      metrics: [NUM_METRIC],
+      viz_type: 'line',
+      time_range: '100 years ago : now',
+    };
+
+    cy.visitChartByParams(JSON.stringify(formData));
+    cy.verifySliceSuccess({ waitAlias: '@postJson' });
+
+    cy.get('[data-test=time-range-trigger]')
+      .click()
+      .then($range => {
+        const firstRange = $range.text();
```

Comment:
You can probably move this variable out of the `then` wrapper with `let` to avoid nesting.

Reply:
I cannot access text from element without using then. I think in Cypress it is impossible.